test/rxreact.test.tsx

```diff
@@ -337,6 +344,15 @@ describe("withViewModel", () => {
       expect(rendered.find("#subject-derived").text()).toContain("oranges");
     });
 
+    it("unsubscribes to local subscriptions on unmount", () => {
+      expect(onUnsubscribe).not.toHaveBeenCalled();
+      rendered.unmount();
+      expect(onUnsubscribe).toHaveBeenCalled();
+
+      // To work with the afterEach block
+      rendered.mount();
```

Comment:
This seemed a little hack-y, but otherwise the `afterEach(() => rendered.unmount())` would throw an error that it was already unmounted